requirements-docs.txt

```diff
@@ -79,7 +79,6 @@ pyrfc3339==1.1            # via -r requirements.txt, acme
 python-dateutil==2.8.1    # via -r requirements.txt, alembic, arrow, botocore
```

Comment:
Note that I chose not to completely regenerate the file at this time - should I? The next time dependabot updates in, I expect significant changes as most of the dependencies will have been dropped. 

I did try building locally as follows:
```
pip-compile --no-index --output-file=requirements-docs.txt requirements-docs.in
make dev-docs
cd docs
make html
```

And it seems to build just fine without any of the removed dependencies.